parisienne/block-template-parts/header.html

```diff
@@ -1,7 +1,7 @@
 <!-- wp:group {"align":"full"} -->
 <div class="wp-block-group alignfull"><div class="wp-block-group__inner-container"><!-- wp:columns {"align":"full"} -->
-<div class="wp-block-columns alignfull"><!-- wp:column {"width":20} -->
-<div class="wp-block-column" style="flex-basis:20%"><!-- wp:navigation {"className":"is-style-dark"} -->
+<div class="wp-block-columns alignfull"><!-- wp:column {"width":25} -->
```

Comment:
These changes seem unrelated, right? Just want to make sure they're intentional.

Reply:
Unrelated! Going to pull them out of this PR.